packages/dara-core/js/shared/utils/clean-session-cache.tsx

```diff
@@ -6,13 +6,21 @@
  */
 function cleanSessionCache(sessionToken: string): void {
     const sessionKeys = Object.keys(sessionStorage);
+    const localKeys = Object.keys(localStorage);
```

Comment:
Minor but this could be simplified if we just loop through the two storages, e.g.

```tsx
for (const storage of [sessionStorage, localStorage]) {
 const keys = Object.keys(storage);
 ...
}
```